wire-ios-transport/Source/PushChannel/StarscreamPushChannel.swift

```diff
@@ -257,6 +257,8 @@ extension StarscreamPushChannel: WebSocketDelegate {
             break
         case .cancelled:
             onClose()
+        case .peerClosed:
+            onClose()
```

Comment:
Not sure if we should call this or just do nothing

Reply:
🤷‍♂️